Libraries/Modal/Modal.js

```diff
@@ -65,11 +65,11 @@ Modal.propTypes = {
 var styles = StyleSheet.create({
   modal: {
     position: 'absolute',
+    left:0,top:0,right:0,bottom:0
```

Comment:
This doesn't seem required. The element with style `modal` shouldn't be taking up room in the layout tree.